components/engine_traits/src/options.rs

```diff
@@ -67,6 +67,32 @@ pub enum SeekMode {
     Prefix,
 }
 
+#[derive(Copy, Clone)]
+pub enum ReadTier {
+    ReadAllTier,
+    BlockCacheTier,
+    PersistedTier,
+    MemtableTier,
+}
+
+impl Default for ReadTier {
+    fn default() -> ReadTier {
+        ReadTier::ReadAllTier
+    }
+}
+
+impl ReadTier {
+    // see [RocksDB option API](https://github.com/facebook/rocksdb/blob/master/include/rocksdb/options.h)
+    pub fn to_tier(&self) -> i32 {
```

Comment:
Should this be moved to `engine_rocks`? It seems not so appropriate to put the function that converts `ReadTier` to Rocksdb read tier value in the generic `engine_traits` crate.